client/src/stores/user.ts

```diff
@@ -0,0 +1,28 @@
+import { defineStore } from 'pinia'
+import { useStorage } from '@vueuse/core'
+import { computed } from 'vue'
+import type { User } from '@/types/user'
+
+export const useUserStore = defineStore('user', () => {
+  //Import userData from types/user
```

Comment:
So I just wanna say the next line is import from ```types/user.ts```, if it does not make sense I can just delete it.

Reply:
Ah I see. I'd be more accurate to say something like "the type below is imported from types/user", but that's not really necessary to say since you can just look at the import statement above.